flask_ckeditor/utils.py

```diff
@@ -15,3 +20,20 @@ def random_filename(old_filename):
     ext = os.path.splitext(old_filename)[1]
     new_filename = uuid.uuid4().hex + ext
     return new_filename
+
+
+def cleanify(text, *, allow_tags=None):
+    """clean the input from client, this function rely on bleach,
+
+
+    Args:
+        text (str): input
+        allow_tags (Iterable[str], optional): if you don't want to use default `allow_tags`
+            you can provide a Iterable which include html tag string like ['a', 'li',...]
+    """
+    if allow_tags:
+        return bleach.linkify(bleach.clean(text, tags=allow_tags))
+    default_allowed_tags = {'a', 'abbr', 'b', 'blockquote', 'code',
+                            'em', 'i', 'li', 'ol', 'pre', 'strong', 'ul',
+                            'h1', 'h2', 'h3', 'h4', 'h5', 'p'}
+    return bleach.linkify(bleach.clean(text, tags=default_allowed_tags))
```

Comment:
You can simplify the code to this:

```suggestion
    default_allowed_tags = {'a', 'abbr', 'b', 'blockquote', 'code',
                            'em', 'i', 'li', 'ol', 'pre', 'strong', 'ul',
                            'h1', 'h2', 'h3', 'h4', 'h5', 'p'}
    return bleach.linkify(bleach.clean(text, tags=allow_tags or default_allowed_tags))
```

Reply:
cool， i got it ！

Reply:
**Thank you for your patient review !** 
i has fixed those questions .